src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -1478,7 +1479,8 @@ private void checkAchievements()
         // the colonies size
         final int size = this.citizens.size();
 
-        if (size >= ModAchievements.ACHIEVEMENT_SIZE_SETTLEMENT)
+        //todo check those later again
+        /*if (size >= ModAchievements.ACHIEVEMENT_SIZE_SETTLEMENT)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)